hooks/useMapLayer.ts

```diff
@@ -33,6 +33,8 @@ export const useMapLayer = (center_id: AvalancheCenterID) => {
     },
     {
       enabled: !!center_id,
+      staleTime: 24 * 60 * 60 * 1000, // don't bother re-fetching for one day (in milliseconds)
+      cacheTime: Infinity, // hold on to this cached data forever
```

Comment:
if we're going to cache this request as aggressively as we are here, then we probably should remove the code from `useMapViewZones` that sets danger levels based on the results of this query (line 39). otherwise there's a strong chance that we display invalid data on the map. https://github.com/stevekuznetsov/avalanche-forecast/blob/05f5a4a8343e9cb70f12cf8ccee5e3b649e9049b/hooks/useMapViewZones.ts#L33-L42

Reply:
Does that basically just collapse `useMapViewZones` into `useAvalancheForecastFragments` then? Are there values of the caching tunables that you'd think better serve us? Do we need to make the decision today?

Reply:
> useQuery has the ability to both return cached data immediately and fetch fresh data

From my read of the docs, I think that's what we do with this PR - the stale time determines how often that background fetch happens, and the other param determines how long we keep the thing in the cache in general.